lib/components/viewers/styled.js

```diff
@@ -50,19 +55,22 @@ export const StopContainer = styled.div`
   are shown when browsers don't calculate 100% sensibly */
   padding: 15px 0 100px;
 `
-export const Stop = styled.a`
+export const StopLink = styled.a`
   cursor: pointer;
   color: ${(props) => props?.textColor + 'da' || '#333'};
+
+  &:hover {
+    color: ${(props) => props?.textColor || '#23527c'};
+  }
+`
+export const Stop = styled.li`
+  cursor: pointer;
```

Comment:
Does the lack of `href` messes up the cursor?

Reply:
That must be it, yes removing this line keeps the cursor from changing. I can remove the `cursor: pointer` from line 59 though